org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java

```diff
@@ -1242,6 +1271,66 @@ final private String getThisSymProperty(final String symbology,
     }
     return null;
   }
+  
```

Comment:
we don't use the object returned by this method. We no longer have to keep a list of what has been added.  This logic can be removed.

Reply:
done